packages/ui/src/App.tsx

```diff
@@ -53,6 +54,8 @@ const App = () => {
             <Route path="/callback/:connector" element={<Callback />} />
             <Route path="/social-register/:connector" element={<SocialRegister />} />
             <Route path="/:type/:method/passcode-validation" element={<Passcode />} />
+            <Route path="/404" element={<NotFound />} />
```

Comment:
Is this for debug only?

Reply:
nope， for all  the unmatched URL path show not found page